sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -390,6 +390,248 @@ pi_result piKernelGetInfo(pi_kernel Kernel, pi_kernel_info ParamName,
                                 ParamValueSizeRet);
 }
 
+pi_result piextMemUnsampledImageHandleDestroy(pi_context context,
+                                              pi_device device,
+                                              pi_image_handle handle) {
+
+  assert(context != nullptr);
+  assert(device != nullptr);
+  assert(handle != 0);
```

Comment:
Release build will fail due to `-WError` and unused parameters. I'd suggest using `[[maybe_unused]]` on function parameters to fix.

Reply:
Added `[[maybe_unused]]` as necessary.